content/200-concepts/300-more/050-environment-variables/index.mdx

```diff
@@ -73,14 +84,6 @@ You can set the `DATABASE_URL` in your `.env` file:
 DATABASE_URL=postgresql://test:test@localhost:5432/test?schema=public
 ```
 
-Variables can be expanded using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
-
-```
-# .env
-DATABASE_URL=postgresql://test:test@localhost:5432/test
-DATABASE_URL_WITH_SCHEMA=${DATABASE_URL}?schema=public
-```
```

Comment:
Where did this go? 
This is some information about something you can do just in the `.env` files and is quite important, as it let's people compose environment variables. So this should get its own headline probably where we explain the .env file functionality, and also be cross linked in the main article about env vars.

Reply:
I didn't mean to remove this actually, I think I was planning on adding it back but forgot. It's now been added back under its own section